packages/web-app-files/src/composables/resourcesViewDefaults/useResourcesViewDefaults.ts

```diff
@@ -76,6 +69,18 @@ export const useResourcesViewDefaults = <T, TT, TU extends any[]>(
   const currentTilesSize = computed((): string => String(currentTilesSizeQuery.value))
   const viewSize = useViewSize(currentTilesSize)
 
+  const fields = computed((): SortField[] => {
+    if (unref(viewMode) === ViewModeConstants.tilesView.name) {
+      return determineResourceTilesSortFields(unref(storeItems)[0])
+    }
+    return determineResourceTableSortFields(unref(storeItems)[0])
+  })
```

Comment:
eeeks ... why is this needed now?

Reply:
Because the Tile view has different sorting fields then the table. Although I'm still debating if we keep one `determineSortFields` and let that handle the fields.